src/System.Management.Automation/utils/ClrFacade.cs

```diff
@@ -209,7 +210,7 @@ internal static Encoding GetOEMEncoding()
             if (s_oemEncoding == null)
             {
 #if UNIX        // PowerShell Core on Unix
-                s_oemEncoding = GetDefaultEncoding();
+                s_oemEncoding = PowerShellEncoding.GetDefaultEncoding();
 #elif CORECLR   // PowerShell Core on Windows
```

Comment:
The discussion was in #3248 and fixed in #3467. Maybe it should be reviewed again. In short we returned behavior Windows PowerShell.